blamo/lib/Unit/units.dart

```diff
@@ -205,7 +205,53 @@ Widget getScaffold(List<Unit> units){
   }
 
   void _onTileLongClicked(int i) async {
+    String result;
+    result = await showDialog(
+        context: context,
+        builder: (context) => AlertDialog(
+          title: Text("Are you sure you want to delete ${currentState.unitList[i]}?"),
```

Comment:
What do you think about showing the unit bounds as displayed on the units page instead of the unit name? My thought is that a unit name doesn't necessarily correlate to what the user is trying to delete, so validation could be a lil sketchy in that regard